openlibrary/templates/search/sort_options.html

```diff
@@ -19,6 +19,7 @@
        sort_options = [
          { 'sort': 'relevance', 'name': _("Relevance"), 'ga_key': 'Relevance' },
          { 'sort': 'editions', 'name': _("Most Editions"), 'ga_key': 'Editions' },
+         { 'sort': 'title', 'name': _("Alphabetical"), 'ga_key': 'Alphabetical' },
```

Comment:
```suggestion
         { 'sort': 'title_sort', 'name': _("Alphabetical"), 'ga_key': 'Alphabetical' },
```

This is the field @mheiman added which better removes articles. We will have to add it to the work scheme file as an allowable field, and as a sort field.